### DISCUSSION TITLE
Can a domain be proxied to another host instead of a container?

### DISCUSSION THREAD

**Discussion by marcopaggioro** [nginx-proxy/nginx-proxy#1911] [Q&A] [answered]
Hi,
I understood that it is enough to add the VIRTUAL_HOST environment variable to the containers I want to connect to nginx-proxy.
So test1.domain.com is resolved on the host running the nginx-proxy container. The nginx-proxy redirect all traffic to the container that has VIRTUAL_HOST="test1.domain.com"

I have test2.domain.com that is resolved on the host running the nginx-proxy container but I would like that the traffic will be redirected to another host recognized by an ip. So I haven't the container locally and I can't assign VIRTUAL_HOST.
Can be possibile?

**Comment by tkw1536** [comment, marked as answer]
This isn't support at the moment, although I believe both #1100 and #1862 are PRs for adding the feature. 

A workaround might be to create a new container on the host that only forwards traffic to the external ip, and assign that a `VIRTUAL_HOST`.
